x-pack/plugins/data_visualizer/public/application/common/components/multi_select_picker/multi_select_picker.tsx

```diff
@@ -32,18 +33,16 @@ const SELECT_PICKER_HEIGHT = '250px';
 
 const NoFilterItems = () => {
   return (
-    <div className="euiFilterSelect__note">
-      <div className="euiFilterSelect__noteContent">
-        <EuiIcon type="minusInCircle" />
-        <EuiSpacer size="xs" />
-        <p>
-          <FormattedMessage
-            id="xpack.dataVisualizer.multiSelectPicker.NoFiltersFoundMessage"
-            defaultMessage="No filters found"
-          />
-        </p>
-      </div>
-    </div>
+    <EuiSelectableMessage>
```

Comment:
@breehall what are the plans for switching out the deprecated `EuiFilterSelectItem` for `EuiSelectable`, as is still being used in our file here?

Reply:
There's a few too many usages in Kibana for EUI to tackle every single one of these ourselves, so we left the inline comments in hopes that Kibana teams would grab the deprecation conversions in their own time. It should hopefully be a fairly straightforward conversion, and if needed you can reference our docs example: https://elastic.github.io/eui/#/forms/filter-group#multi-select

As I mentioned in another comment, the final removal of `EuiFilterSelectItem` is still 6months+ out, so hopefully plenty of time for y'all - but please let us know if we can be a resource for this!